.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	parserOptions: {
+		project: './tsconfig.json',
+		tsconfigRootDir: __dirname,
+		ecmaVersion: 12,
+		sourceType: 'module',
+	},
+	extends: ['web3-base/ts'],
```

Comment:
Is it referring to [this](https://github.com/ChainSafe/web3.js/tree/nh/4341-ts-setup/tools/eslint-config-web3-base) with correct package name ?

Reply:
Yes its the `eslint` convention when you extend it tries to prefix `eslint-config` automatically. So its actually extending `eslint-config-web3-base/ts`